src/main/definitions/case.ts

```diff
@@ -40,6 +40,8 @@ export interface Case {
   claimantRepresentedQuestion?: YesOrNo;
   caseType?: CaseType;
   caseTypeId?: CaseTypeId;
+  gender?: string;
```

Comment:
I don't think we need this property. claimantSex should be the value we reference

Reply:
@olyode

Reply:
Yes referencing claimantSex, gender is now removed